src/components/Widgets/useWidgetTheme.ts

```diff
@@ -21,10 +21,10 @@ export const useWidgetTheme = (): PartnerTheme => {
     config: {
       appearance: theme.palette.mode,
       theme: {
-        //todo: fix typography
-        // typography: {
-        //   fontFamily: theme.typography,
-        // },
+        // @ts-expect-error
```

Comment:
What error do we have here?

Reply:
@chybisov 
![image](https://github.com/user-attachments/assets/3d9bfdab-b357-4e41-8916-d4ae9d5ab724)

It´s complaining because of our custom typographies which is likely not included in a partner-theme.
> brandHeaderXLarge, bodyXLarge, bodyLargeStrong, bodyLarge, and 6 more